test/modules/TestResultCoordinator/report/TestSummary.cs

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace TestResultCoordinator.Report
+{
+    using System.Linq;
+
+    class TestSummary
+    {
+        internal TestSummary(ITestResultReport[] testResultReports)
+        {
+            this.TestResultReports = testResultReports;
+            this.AreAllTestsPassed = testResultReports.All(r => r.IsPassed);
```

Comment:
suggestion: name this IsPassed, I think it should be clear since it is in TestSummary that it means all test passed